internal/driver/cli.go

```diff
@@ -52,7 +52,7 @@ func parseFlags(o *plugin.Options) (*source, []string, error) {
 	flagSymbolize := flag.String("symbolize", "", "Options for profile symbolization")
 	flagBuildID := flag.String("buildid", "", "Override build id for first mapping")
 	flagTimeout := flag.Int("timeout", -1, "Timeout in seconds for fetching a profile")
-	flagAddComment := flag.String("add_comment", "", "Annotation string to record in the profile")
+	flagAddComment := flag.StringList("add_comment", "", "Annotation string to record in the profile")
```

Comment:
Does adding multiple comments work if you test this end-to-end manually? I don't think StringList is currently properly implemented - [code](https://github.com/google/pprof/blob/c488b8fa1db3fa467bf30beb5a1d6f4f10bb1b87/internal/driver/flags.go#L49).

Reply:
I've created a basic implementation.